.github/workflows/pages.yml

```diff
@@ -0,0 +1,93 @@
+name: 📕 Deploy docs site to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["master"]
```

Comment:
What would be effect of enabling it on all branches?

Reply:
It depends on if you want to do PR's for docs. Like if you want to edit docs in develop then PR it to master. Since it's just documentation and not source code, it can just be edited on master and pushed to master.

Reply:
Let's start with master only version.